### PR TITLE
Upgrade ZFS from 0.8.5 to 2.1.14

### PR DESCRIPTION
**Pull request by gthvn1**

### CODE REVIEW

File: SPECS/zfs-kmod.spec [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
 Name:           %{module}-kmod
 
-Version:        0.8.5
+Version:        2.1.14
 Release:        2%{?dist}
```
Comment (stormi) [inline SPECS/zfs-kmod.spec:55]: Should it be back to 1?

Comment (gthvn1) [reply SPECS/zfs-kmod.spec:55]: Mmmm yes I think that would be logical. I will do that.

Comment (gthvn1) [reply SPECS/zfs-kmod.spec:55]: I will do the same for ZFS

**Comment by gthvn1** [comment]
I will update upstream instead of this PR

**Pull request closed without merging**